hw/audio/ac97.c

```diff
@@ -1112,9 +1167,11 @@ static void transfer_audio (AC97LinkState *s, int index, int elapsed)
             if (r->civ == r->lvi) {
                 dolog ("Underrun civ (%d) == lvi (%d)\n", r->civ, r->lvi);
 
-                new_sr |= SR_LVBCI | SR_DCH | SR_CELV;
+                //new_sr |= SR_LVBCI | SR_DCH | SR_CELV;
```

Comment:
I don't understand why this is necessary. On hardare it only seems to set `SR_LVBCI`. However, if I do that, the interrupts won't be triggered anymore and audio drains.